pkg/registry/core/persistentvolumeclaim/strategy_test.go

```diff
@@ -76,15 +76,13 @@ func TestDropConditions(t *testing.T) {
 		},
 	}
 
-	for _, enabled := range []bool{true, false} {
+	for _, enabled := range []bool{true} {
```

Comment:
I think this loop can be removed completely

Reply:
fixed